src/helpers/merging.py

```diff
@@ -0,0 +1,51 @@
+from constants import OUTPUT_COLUMNS
+
+NUMERIC_COLUMNS = ['Pop Tested',
+                   'Pop Tested Positive',
+                   'Pop Tested Negative',
+                   'Pop Deaths',
+                   'Staff Tested',
+                   'Staff Tested Positive',
+                   'Staff Tested Negative',
+                   'Staff Deaths']
+TEXT_COLUMNS = ['Source',
+                'Notes']
+
+assert set(NUMERIC_COLUMNS).issubset(set(OUTPUT_COLUMNS))
+assert set(TEXT_COLUMNS).issubset(set(OUTPUT_COLUMNS))
+
+
+def get_value(row, column):
```

Comment:
This function is a bit odd in that it wants to return an integer but if it can't return an integer it returns an empty string. Docstrings will help readability, as noted previously, but I wonder if there's a clean way to update this to return a consistent type? Maybe by returning `None` if there's an error instead of an empty string?

Reply:
Agreed. Updated to return `None`.